### DISCUSSION TITLE
Connect Google Drive in Latest Colab?

### DISCUSSION THREAD

**Discussion by jorymo** [TheLastBen/fast-stable-diffusion#1894] [Q&A]
Maybe a dumb question, but I noticed the Automatic colab recently added that cell at the top to connect a google drive. Assuming it's like older versions with an option to mount my google drive, what exactly would I put there?

**Comment by TheLastBen** [comment]
if you don't use a shared drive, leave the box empty and run the cell

**Reply by Omenizer** [reply]
How do we use the shared drive? I put a link to a shared folder there but it didn't work.

**Reply by TheLastBen** [reply]
not shared folder, shared drive https://support.google.com/a/users/answer/7212025?hl=en

**Reply by GoofyAi** [reply]
i have a shared drive. how can i share the drive link with the colab

**Reply by TheLastBen** [reply]
you simply input the name of the shared drive, not a path just the name
